pkg/tools/callback/server.go

```diff
@@ -21,12 +21,20 @@ import (
 	"context"
 	"fmt"
 	"net"
+	"strings"
 	"sync"
 
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/metadata"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/grpcutils"
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+)
+
+const (
+	callbackPrefix = "callback:"
```

Comment:
```suggestion
	callbackPrefix = "callback"
```

Reply:
I do not think it is required, it could be any prefix. I've choose one with ':'